### DISCUSSION TITLE
Docs for nucleo-matcher

### DISCUSSION THREAD

**Discussion by max397574** [helix-editor/nucleo#19] [General]
As far as I can tell currently there are no docs for nucleo matcher. I'd be interested in using this crate in one of my projects. for that i'd need docs on how I can use it though. I mean I could also read the source but I think the crate would be much more accessible with docs.

**Comment by pascalkuthe** [comment]
Ofcourse nucleo-matcher has docs:

https://docs.rs/nucleo-matcher/0.2.0/nucleo_matcher/

What made you think it didn't?

**Reply by max397574** [reply]
that there was just the nucleo readme on crates.io
